presto-main/src/main/java/com/facebook/presto/operator/aggregation/TypedSet.java

```diff
@@ -147,6 +147,11 @@ public boolean addNonNull(Block block, int position)
         return false;
     }
 
+    public boolean isContainsNullElement()
+    {
+        return containsNullElement;
+    }
+
```

Comment:
Revert this change as well. We don't need it.

Reply:
added back this since for O(N log N) used TypedSet as solution

Reply:
You can just name the function `containsNullElements`